app/models/user.rb

```diff
@@ -60,6 +60,12 @@ class User < ApplicationRecord
           class_name: 'InPersonEnrollment', foreign_key: :user_id, inverse_of: :user,
           dependent: :destroy
 
+  has_one :current_profile,
+          -> { order(created_at: :desc) },
+          class_name: 'Profile',
+          dependent: :destroy,
+          inverse_of: :user
```

Comment:
I'd consider putting this next to `has_many :profiles` ? and maybe we ditch the `dependent: :destroy` since it's redindant with the one on that relation?

Reply:
Not having it upsets the `Rails/HasManyOrHasOneDependent` cop. I kind of agree with the cop here because I like having it on there in case something changes somehow in the future.